planner/core/common_plans.go

```diff
@@ -187,6 +187,20 @@ type Execute struct {
 	Plan          Plan
 }
 
+// Check if result of GetVar expr is BinaryLiteral
+// Because GetVar use String to represent BinaryLiteral, here we need to convert string back to BinaryLiteral.
+func isGetVarBinaryLiteral(sctx sessionctx.Context, expr expression.Expression) bool {
+	res := false
```

Comment:
The `res` looks redundant in this function, we can directly return `dt.Kind() == types.KindBinaryLiteral` of false.